apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/server/S3DirectoryDaoIT.java

```diff
@@ -234,23 +231,24 @@ public void testGetObjectMetaDataForMissingFile() throws Exception {
   /**
    * Upload the string as a "file" to the s3 bucket and return the ETag assigned to it by S3.
    *
-   * @param s3Client the {@link AmazonS3} client to use
+   * @param s3Client the {@link S3Client} client to use
    * @param bucket the bucket to receive the file
    * @param objectKey the key for the object
    * @param fileData a string uploaded as a file
    * @return eTag assigned to the file by S3
    * @throws IOException pass through if anything fails
    */
-  private String uploadFileToBucket(
-      AmazonS3 s3Client, Bucket bucket, String objectKey, String fileData) throws IOException {
+  private String uploadFileToString(
+      S3Client s3Client, String bucket, String objectKey, String fileData) throws IOException {
```

Comment:
How does one upload a file to a string?  I think the name fell victim to a search/replace. :-)